Source/BestProjectEver/Public/GameElements/BPE_SpawnPad.h

```diff
@@ -78,6 +81,23 @@ class BESTPROJECTEVER_API ABPE_SpawnPad : public AActor
 	/** for visual effect when an actor is spawned */
 	UPROPERTY(EditAnywhere, Category= "Spawner")
 	UCurveFloat* RingMovementCurve;
+
+	//------------------------------------------------------------------------------------------------------------------
+
+	/** Delay for reset emissive of button material */
+	UPROPERTY(EditDefaultsOnly, Category="Materials")
+	float SetButtonEmissiveDelay;
```

Comment:
You're naming this field as a setter when having it start with Set. Maybe something like `ButtonEmissiveResetDelay` would sound more like property rather than a function? Or `ButtonEmissiveDuration`? Avoid having variable names starting with verbs.

Reply:
![image](https://user-images.githubusercontent.com/85327912/174651242-137bce92-4128-4960-9d46-5ff2b9f6947a.png)